File: Python/UberVan.py
from car import Car 

from uberadvanced import UberAdvanced

class UberVan (UberAdvanced):
    def __init__ (self, id, license, driver, passengers, typeCarAccepted, seatsMaterialal):
        super().__init__(id, license, driver, passengers, typeCarAccepted, seatsMaterialal)

    def setPassengers (self, passengers):
        if (passengers >= 6):
            self._passengers = passengers
        else:
            print("Debe haber 6 o más pasajeros")

File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map

class UberVan extends Car {
    Map<String, ArrayList<String,integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver, Account brand, Account model, Map<String, ArrayList<String,integer>> typeCarAccepted, ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial
    }
}